stdlib/builtins.pyi

```diff
@@ -179,6 +179,9 @@ class super(object):
     @overload
     def __init__(self) -> None: ...
 
+_PositiveInteger = Literal[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25]
```

Comment:
@AlexWaygood, could you answer a couple of my questions?
1. As far I understand the reason to have `_PositiveInteger`/`_NegativeInteger` is because we can't create "all positive/negative ints" types, right? So by defining a Literal we have at least some type inference for small exponents.
2. Why did you choose to stop at 25/-20? My only guess now is that it fits in one line according to the formatting rules in typeshed 😅
3. Why wasn't 0 exponent included in `_PositiveInteger`? I know it wouldn't be strictly "positive" integer then but 2**0 produces `int` in Python and currently it is `Any` according to typeshed.

Thank you!

Reply:
1. Yup, you're quite correct! There's no way (not yet, anyway) of expressing "positive integer" in a general way, but if you're calling it with literal types, they're likely to be quite small -- and this hack is better than nothing :)
2. Yup, once again spot on! I can't remember whether this is exactly the maximum line length black will allow under the settings typeshed has in `pyproject.toml` -- but it's close to it, and they're nice round numbers ;)
3. Um... good catch! No reason at all! Feel free to submit a PR correcting my oversight! 🙂